### ISSUE TITLE
displayName Clarification 

### ISSUE THREAD

**Issue by bcomnes** [af/JSnoX#13]
Whats the proper way to populate `displayName` when using JSnoX?  I didn't (or maybe missed) any mention of this?  It doesn't seem to happen automatically like with JSX.

**Comment by af** [comment]
Hi,
I believe Babel [infers it from the filename](https://github.com/babel/babel/issues/1331) if you're using Babel and `React.createClass()`. JSnoX doesn't infer it automatically so you may want to declare `displayName` manually if you're not transpiling your code. If you're transpiling for ES6 classes, you can use the Babel transform in the link above. 

Hope that helps– if you find any other tricks to have it populated automatically please let me know!

**Comment by bcomnes** [comment]
Darn, it seems that link is 404ing.  

Yeah, was trying to see how far I could get without transpiling to keep things simple.  I figured that it was inferred by some kind of inflection of the variable name you assign the class name too, but it looks like I have some reading to do.  

Do you personally transpile you react projects in most cases (with or without JS(no)X)?

**Comment by af** [comment]
That's weird– here's [a cached version](https://webcache.googleusercontent.com/search?q=cache:https://github.com/babel/babel/issues/1331) of that babel issue.

I went without transpilation for a while, and at that point I was specifying the displayName manually (I used a snippet in my editor so it wasn't too cumbersome). I'm getting pretty hooked on ES6 now though, so I'm using ES6 class components (without JSX). I haven't gotten back in the habit of adding `displayName`, though I probably should.
